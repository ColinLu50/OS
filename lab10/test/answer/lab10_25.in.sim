7870
520
358
292
356
290
