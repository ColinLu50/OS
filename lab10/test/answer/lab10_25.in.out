FCFS
65 -> 167 -> 67 -> 289 -> 264 -> 161 -> 115 -> 39 -> 150 -> 112 -> 20 -> 47 -> 248 -> 239 -> 237 -> 156 -> 218 -> 287 -> 251 -> 13 -> 133 -> 228 -> 162 -> 191 -> 250 -> 42 -> 191 -> 267 -> 142 -> 117 -> 1 -> 223 -> 258 -> 117 -> 156 -> 75 -> 49 -> 263 -> 290 -> 187 -> 160 -> 72 -> 261 -> 17 -> 60 -> 170 -> 125 -> 247 -> 257 -> 39 -> 134 -> 166 -> 51 -> 83 -> 204 -> 162 -> 151 -> 79 -> 6 -> 6 -> 293 -> 87 -> 127 -> 40 -> 4 -> 82 -> 9 -> 286 -> 119 -> 2 -> 73 -> 290 -> 267 -> 84 -> 127 -> 60 -> 133 -> 57 -> 31 -> 28 -> 73 -> 289 -> 277 -> 282 -> 239 -> 125
7870
SSTF
65 -> 67 -> 72 -> 73 -> 73 -> 75 -> 79 -> 82 -> 83 -> 84 -> 87 -> 112 -> 115 -> 117 -> 117 -> 119 -> 125 -> 125 -> 127 -> 127 -> 133 -> 133 -> 134 -> 142 -> 150 -> 151 -> 156 -> 156 -> 160 -> 161 -> 162 -> 162 -> 166 -> 167 -> 170 -> 187 -> 191 -> 191 -> 204 -> 218 -> 223 -> 228 -> 237 -> 239 -> 239 -> 247 -> 248 -> 250 -> 251 -> 257 -> 258 -> 261 -> 263 -> 264 -> 267 -> 267 -> 277 -> 282 -> 286 -> 287 -> 289 -> 289 -> 290 -> 290 -> 293 -> 60 -> 60 -> 57 -> 51 -> 49 -> 47 -> 42 -> 40 -> 39 -> 39 -> 31 -> 28 -> 20 -> 17 -> 13 -> 9 -> 6 -> 6 -> 4 -> 2 -> 1
520
SCAN
65 -> 60 -> 57 -> 51 -> 49 -> 47 -> 42 -> 40 -> 39 -> 31 -> 28 -> 20 -> 17 -> 13 -> 9 -> 6 -> 4 -> 2 -> 1 -> 0 -> 67 -> 72 -> 73 -> 75 -> 79 -> 82 -> 83 -> 84 -> 87 -> 112 -> 115 -> 117 -> 119 -> 125 -> 127 -> 133 -> 134 -> 142 -> 150 -> 151 -> 156 -> 160 -> 161 -> 162 -> 166 -> 167 -> 170 -> 187 -> 191 -> 204 -> 218 -> 223 -> 228 -> 237 -> 239 -> 247 -> 248 -> 250 -> 251 -> 257 -> 258 -> 261 -> 263 -> 264 -> 267 -> 277 -> 282 -> 286 -> 287 -> 289 -> 290 -> 293
358
C-SCAN
65 -> 60 -> 57 -> 51 -> 49 -> 47 -> 42 -> 40 -> 39 -> 31 -> 28 -> 20 -> 17 -> 13 -> 9 -> 6 -> 4 -> 2 -> 1 -> 0 -> 294 -> 293 -> 290 -> 289 -> 287 -> 286 -> 282 -> 277 -> 267 -> 264 -> 263 -> 261 -> 258 -> 257 -> 251 -> 250 -> 248 -> 247 -> 239 -> 237 -> 228 -> 223 -> 218 -> 204 -> 191 -> 187 -> 170 -> 167 -> 166 -> 162 -> 161 -> 160 -> 156 -> 151 -> 150 -> 142 -> 134 -> 133 -> 127 -> 125 -> 119 -> 117 -> 115 -> 112 -> 87 -> 84 -> 83 -> 82 -> 79 -> 75 -> 73 -> 72 -> 67
292
LOOK
65 -> 60 -> 57 -> 51 -> 49 -> 47 -> 42 -> 40 -> 39 -> 31 -> 28 -> 20 -> 17 -> 13 -> 9 -> 6 -> 4 -> 2 -> 1 -> 67 -> 72 -> 73 -> 75 -> 79 -> 82 -> 83 -> 84 -> 87 -> 112 -> 115 -> 117 -> 119 -> 125 -> 127 -> 133 -> 134 -> 142 -> 150 -> 151 -> 156 -> 160 -> 161 -> 162 -> 166 -> 167 -> 170 -> 187 -> 191 -> 204 -> 218 -> 223 -> 228 -> 237 -> 239 -> 247 -> 248 -> 250 -> 251 -> 257 -> 258 -> 261 -> 263 -> 264 -> 267 -> 277 -> 282 -> 286 -> 287 -> 289 -> 290 -> 293
356
C-LOOK
65 -> 60 -> 57 -> 51 -> 49 -> 47 -> 42 -> 40 -> 39 -> 31 -> 28 -> 20 -> 17 -> 13 -> 9 -> 6 -> 4 -> 2 -> 1 -> 293 -> 290 -> 289 -> 287 -> 286 -> 282 -> 277 -> 267 -> 264 -> 263 -> 261 -> 258 -> 257 -> 251 -> 250 -> 248 -> 247 -> 239 -> 237 -> 228 -> 223 -> 218 -> 204 -> 191 -> 187 -> 170 -> 167 -> 166 -> 162 -> 161 -> 160 -> 156 -> 151 -> 150 -> 142 -> 134 -> 133 -> 127 -> 125 -> 119 -> 117 -> 115 -> 112 -> 87 -> 84 -> 83 -> 82 -> 79 -> 75 -> 73 -> 72 -> 67
290
